src/PluginsManager/Microsoft.Performance.Toolkit.Plugins.Runtime/Manager/IPluginsManager.cs

```diff
@@ -112,9 +117,12 @@ public interface IPluginsManager
         ///     Signals that the caller wishes to cancel the operation.
         /// </param>
         /// <returns>
-        ///     A collection of installed plugins.
+        ///     A <see cref="InstalledPluginsResults"/> instance containing all valid and invalid installed plugins.
         /// </returns>
-        Task<IReadOnlyCollection<InstalledPlugin>> GetInstalledPluginsAsync(CancellationToken cancellationToken);
+        /// <exception cref="OperationCanceledException">
+        ///     The operation was canceled.
+        /// </exception>
```

Comment:
I assume this can throw other exceptions, like if the registry is corrupt and what not?